general/pages/index.vue

```diff
@@ -72,12 +72,19 @@ export default {
 
     try {
       const response = await getStrapiSingleType(singleTypeName, populateParams);
-      const title = response.data.data.attributes.sections[0].title || process.env.VUE_ONTOLOGY_NAME;
+      if(response?.data?.data?.attributes?.sections == null)
+      {
+        console.error(`Page data(sections) is not recognized in the response from the server.
+        Error occurred while rendering page ${singleTypeName}.\n
+        Current server response:\n${response}`);
```

Comment:
response in string will display as [object Object] (this is screen from local test):
![image](https://user-images.githubusercontent.com/87621210/208879265-427bffd6-e8f5-446f-871d-602cf99097d0.png)

the response object can be passed as a second argument of console.error. So the line would look like this:
`Current server response:\n`, response);`

then the whole object will be printed in json format:
![image](https://user-images.githubusercontent.com/87621210/208879677-3823b766-5c01-44a4-899d-20ff85967fa6.png)